core/envs/generic_env.py

```diff
@@ -163,4 +161,4 @@ def _gather_imus_frame(self) -> torch.Tensor:
 
         return torch.cat(
```

Comment:
maybe a comment explain why cpu. We know having worked at it? but fresh eyes would not.

Reply:
True that!